man/homectl.xml

```diff
@@ -506,6 +506,40 @@
         <xi:include href="version-info.xml" xpointer="v247"/></listitem>
       </varlistentry>
 
+    <varlistentry>
+        <term><option>--blob=</option><replaceable>PATH</replaceable></term>
+        <term><option>-b</option> <replaceable>PATH</replaceable></term>
+        <term><option>--blob=</option><replaceable>FILENAME</replaceable>=<replaceable>PATH</replaceable></term>
+        <term><option>-b</option> <replaceable>FILENAME</replaceable>=<replaceable>PATH</replaceable></term>
```

Comment:
i'd probably use `:` as a separator here. feels so weir dhaving two `=` in the same option. but up to you, either is fine

Reply:
`homectl` already has `--rlimit=LIMIT=VALUE`, so I chose `=` for consistency. I also looked a bit through the codebase, and couldn't find `:` being used anywhere in the way I use it here. Conceptually, I am assigning the filename to the (contents of) the file. `:` seems to be used mostly for ranges, or for "alternative" values (i.e. soft vs hard rlimit, which I suppose actually does behave somewhat like a range...), which isn't what we're doing here.